target/linux/mvebu/patches-5.4/408-PCI-aardvark-Don-t-touch-PCIe-registers-if-no-card-c.patch

```diff
@@ -0,0 +1,55 @@
+From 70e380250c3621c55ff218cbaf2272830d9dbb1d Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Pali=20Roh=C3=A1r?= <pali@kernel.org>
+Date: Thu, 2 Jul 2020 10:30:36 +0200
+Subject: [PATCH] PCI: aardvark: Don't touch PCIe registers if no card
```

Comment:
Small nitpick, always keep the backported patches in chronological order, even if they touch different files.

Reply:
pci aardvark patches are in same order. I just put DTS changes to 5xx numbers where are other DTS patches.

Reply:
Ahh! Indeed, the web page has hidden the numbers. Anyway we have a file (https://github.com/openwrt/openwrt/blob/master/target/linux/generic/PATCHES) suggesting the numbering of the patches, and all backports should go to 001-100. I know, not all patches are numbered that way, but these are remnants from old times. It would be good to gradually change it to proper outcome.

Reply:
Looks like that file is for generic, not mvebu. Looks like that mvebu patches have different numbers and its clear that DTS patches use 5xx numbers. That is what I followed to not have a mess in numbers.

Reply:
Ok, lets leave it at that, I'll send reordering patch after this is accepted.